packages/core-components/src/components/QuestionElement.vue

```diff
@@ -30,23 +30,58 @@
           >
             {{ question }}
           </VCol>
-          <VCol cols="3" class="text-right align-content-center">
+          <VCol cols="3" class="text-right d-flex justify-end align-center">
             <PublishDiffChip
               v-if="editorState.isPublishDiff && element.changeSincePublish"
               :change-type="publishDiffChangeType"
             />
             <VFadeTransition>
-              <VBtn
+              <div
                 v-if="!isDisabled && collapsable && (isHovering || expanded)"
-                :color="`secondary-lighten-${expanded ? 3 : 1}`"
-                class="ml-2"
-                density="comfortable"
-                variant="tonal"
-                icon="mdi-delete-outline"
-                size="small"
-                @click.stop="$emit('delete')"
-              />
+                class="d-flex justify-end ga-1"
+              >
+                <VTooltip location="left" open-delay="1000">
```

Comment:
Should this depend on AI enabled flag ? How do we handle context when this is embedded within other elements e.g. question?